docs/migration-guide/to-16.md

```diff
@@ -14,14 +14,11 @@ We've removed the stylistic rules we deprecated in 15.0.0.
 
 You should remove the rules from your configuration object. See the [15.0.0 migration guide](./to-15.md#deprecated-stylistic-rules) for more details.
 
-## Removed support for Node.js 14
+## Removed support for Node.js less than 18.12.0
 
-Node.js 14 has reached end-of-life. We've removed support for it so that we could update some of our dependencies.
```

Comment:
You probably don't have to change this paragraph.

https://github.com/stylelint/stylelint/blob/dca4e789a8c15da76267d322fd4541513fcb6547/package.json#L224

Reply:
This PR changes the engine range to `>=18.12.0`.

Reply:
The previous paragraph said

> Node.js 14 and 16 have reached end-of-life. We've removed support for them so that we could update some of our dependencies.

which would still be accurate.